packages/components/src/components/primitives/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import { IconButton, StyleProps, useStyleConfig } from '@chakra-ui/react';
+import { CloseButtonProps } from './types';
+import { IconX } from '../Icons';
+
+export const CloseButton: FC<CloseButtonProps> = ({ size, ...rest }) => {
+  const styles = useStyleConfig('CloseButton', { size }) as StyleProps;
+  return (
+    <IconButton
+      aria-label={rest['aria-label'] || 'close button'}
+      data-testid="cmpsr.closeButton"
+      icon={<IconX size={size === 's' ? size : 'l'} />}
```

Comment:
About  that line, my first approach was to set the icon size directly in the closeButton styles like this [commit](https://github.com/cmpsr/composer/blob/7ef937867b4f7a0e80fc19ea6082b325245fd661/packages/components/src/theme/ComposerTheme/Components/CloseButton.ts)
``` js
{ 
....
  '& svg': {
        boxSize: '1.5rem',
      },
}
```
but I ended up setting the size by that conditional, validating the prop size that we already have.